File: server/Models/Sculpture.cs
namespace art_site.Models;

public class Sculpture
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public int? Year { get; set; }
    public int? Price { get; set; }
    public bool? IsLarge { get; set; }
    public string CoverImg { get; set; }
}

File: client/src/models/Sculpture.js
export class Sculpture {
    constructor(data) {
        this.name = data.name
        this.coverImg = data.coverImg
        this.createdAt = Date(data.createdAt)
        this.updatedAt = Date(data.updatedAt)
        this.description = data.description
        this.id = data.id
        this.isLarge = data.isLarge
        this.price = data.price
        this.year = data.year
    }
}